cpp/log/cert.cc

```diff
@@ -2,6 +2,7 @@
 #include "log/cert.h"
 
 #include <glog/logging.h>
+#include <util/util.h>
```

Comment:
We try to keep the include ordering alphabetically ordered (and grouped according the Google C++ Style Guide.)